mailet/standard/src/main/java/org/apache/james/transport/mailets/ClamAVScan.java

```diff
@@ -674,7 +677,7 @@ public void service(Mail mail) throws MessagingException {
                     if (isDebug()) {
                         LOGGER.debug("OK (by CLAMD on {})", socket.getInetAddress());
                     }
-                    mail.setAttribute(MAIL_ATTRIBUTE_NAME, "false");
+                    mail.setAttribute(makeAttribute(false));
```

Comment:
did you check that potential other mailets using this attribute are expecting a boolean and not a String?

Reply:
I did, it's only locally used.